### PR TITLE
 Updating bots language code dynamically 

### PR DESCRIPTION
**Pull request by Suraj-Kumar**
### What do you want to achieve?
->

### How was the code tested?

->

### What new thing you came across while writing this code? 
->

### Incase you fixed a bug then please describe the root cause of it? 
->

NOTE: Make sure you're comparing your branch with the correct base branch

### CODE REVIEW

File: webplugin/js/app/km-richtext-markup-1.0.js [hunk 1]
```diff
@@ -143,12 +143,12 @@ getButtonTemplate:function(options,requestType, buttonClass){
 getQuickRepliesTemplate:function(){
     return`<div class="km-cta-multi-button-container">
             {{#payload}}
-                 <button aria-label="{{title}}" title='{{message}}' class="km-quick-replies km-custom-widget-text-color {{buttonClass}} " data-metadata = "{{replyMetadata}}">{{title}}</button>
+                 <button aria-label="{{title}}" title='{{message}}' class="km-quick-replies km-custom-widget-text-color {{buttonClass}} " data-metadata = "{{replyMetadata}}" data-languageCode = "{{updateLanguage}}">{{title}}</button>
```
Comment (devashishmamgain) [inline webplugin/js/app/km-richtext-markup-1.0.js]: @Suraj-Kumar below it is {{action.updateLanguage}} here it is {{updateLanguage}}, both are supposed to be different or same?

Comment (Suraj-Kumar) [reply webplugin/js/app/km-richtext-markup-1.0.js]: both are different depends on JSON being passed.

Comment (devashishmamgain) [reply webplugin/js/app/km-richtext-markup-1.0.js]: is it json format specified by us somewhere or is it through dialogflow, trying to understand what makes this different than getGenericSuggestedReplyButton.
If it is enforced by us then why can't keep it consistent?

Comment (Suraj-Kumar) [reply webplugin/js/app/km-richtext-markup-1.0.js]: getGenericSuggestedReplyButton  was added to support different type of buttons in single message. So the JSON format is slightly different there. once this template is added into mobile SDKs will remove old JSON from docs and promote this one.

**Pull request merged**